lib/pages/login-page.js

```diff
@@ -22,28 +22,27 @@ export default class LoginPage extends AsyncBaseContainer {
 		const passwordSelector = By.css( '#password' );
 
 		await driverHelper.waitTillPresentAndDisplayed( driver, userNameSelector );
-		await this.driver.sleep( 1000 );
+		await driverHelper.waitTillFocused( driver, userNameSelector );
 		await driverHelper.setWhenSettable( driver, userNameSelector, username );
 		await this.driver.sleep( 1000 );
 		await driver.findElement( userNameSelector ).sendKeys( Key.ENTER );
 
-		await this.driver.sleep( 1000 );
-
 		if ( emailSSO === false ) {
+			await driverHelper.waitTillPresentAndDisplayed( driver, passwordSelector );
+			await driverHelper.waitTillFocused( driver, passwordSelector );
```

Comment:
Is it necessary to have both methods, `waitTillPresentAndDisplayed` and `waitTillFocused`? It seems that we are waiting for the same element twice.

Reply:
@Stojdza I tested without `waitTillPresentAndDisplayed` here and on line 24 as well (https://github.com/Automattic/wp-e2e-tests/pull/1698/files#diff-d2cabd92e3f34ad1459c4dd1862e098fR24). It works without it. But I decided to leave it because `waitTillFocused` doesn't check if the element is displayed and `waitTillPresentAndDisplayed` has that check. I think to be extra sure that the element is displayed, it will be good to leave it. What do you think?

>I wanted to try what would happen if I put waitTillFocused inside clickWhenClickable and setWhenSettable but that did not turn out well :D

I did the same but just with `clickWhenClickable` and it was an epic fail.

Reply:
@Stojdza thanks! Does it mean it is good to merge?

Reply:
Yes :)

Reply:
Cool, thanks!